### PR TITLE
[TR] PIM-4785: add error page on pef

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                  | A
| ------------------ | ---
| Migration script?  | N
| Behats             | N
| Specs & Static     | Y
| Changelog updated  | N

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php [hunk 1]
```diff
@@ -46,16 +53,24 @@ public function onKernelException(GetResponseForExceptionEvent $event)
...
-        );
+        if ('html' === $event->getRequest()->get('_format') ||
+            null === $event->getRequest()->get('_format')
+        ) {
+            $content = $this->templating->render(
+                'PimEnrichBundle:Error:base.html.twig',
+                [
+                    'exception'   => $exception,
+                    'status_code' => $response->getStatusCode(),
+                    '_locale'     => $event->getRequest()
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php]: ?

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 2]
```diff
@@ -256,3 +256,7 @@ pim_enrich:
                 title: Add attribute option
                 confirm: Add
                 cancel: Cancel
+
```
Comment (ClementGautier) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: remove this blank line

File: src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php [hunk 3]
```diff
@@ -37,7 +38,13 @@ public function onKernelException(GetResponseForExceptionEvent $event)
     {
         $exception = $event->getException();
 
-        $response = new Response();
+        if ('html' === $event->getRequest()->get('_format') ||
```
Comment (ClementGautier) [inline src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php]: Could we just remove this listener and let the default listener do his job ? http://symfony.com/fr/doc/current/cookbook/controller/error_pages.html

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/error/error.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+define(
+    ['underscore', 'backbone', 'text!pim/template/error/error'],
+    function (_, Backbone, template) {
+        return Backbone.View.extend({
+            template: _.template(template),
+            initialize: function (message, statusCode) {
+                this.message     = message;
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/error/error.js]: indentation error

Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/error/error.js]: declare these properties please

File: src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml [hunk 5]
```diff
@@ -108,10 +108,12 @@ config:
         pim/item/view:                      pimenrich/js/pim-item-view
         pim/i18n:                           pimenrich/js/i18n
         pim/user-context:                   pimenrich/js/user-context
+        pim/error:                          pimenrich/js/error/error
 
         # Security
         pim/security-context:               pimenrich/js/security-context
 
+
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml]: extra line

**Comment by grena** [comment]
GTM once small comments fixed :+1:

**Comment by jmleroux** [comment]
GTM

**Pull request merged**